pkg/api/server/register_images.go

```diff
@@ -638,25 +638,25 @@ func (s *APIServer) registerImagesHandlers(r *mux.Router) error {
 	// summary: Load image
 	// description: Load an image (oci-archive or docker-archive) stream.
 	// parameters:
-	//  - in: query
-	//    name: change
-	//    description: "Apply the following possible instructions to the created image (default []): CMD | ENTRYPOINT | ENV | EXPOSE | LABEL | STOPSIGNAL | USER | VOLUME | WORKDIR.  JSON encoded string"
-	//    type: string
-	//  - in: query
-	//    name: message
-	//    description: Set commit message for imported image
-	//    type: string
-	//  - in: body
-	//    name: request
-	//    description: tarball of container image
-	//    required: true
-	//    schema:
-	//      type: string
+	//   - in: query
+	//     name: change
+	//     description: "Apply the following possible instructions to the created image (default []): CMD | ENTRYPOINT | ENV | EXPOSE | LABEL | STOPSIGNAL | USER | VOLUME | WORKDIR.  JSON encoded string"
+	//     type: string
+	//   - in: query
+	//     name: message
+	//     description: Set commit message for imported image
+	//     type: string
+	//   - in: body
+	//     name: request
+	//     description: tarball of container image
+	//     schema:
+	//       type: string
```

Comment:
If this is tarball that is not uuencoded, I think you want to add the property of "format: binary".